concordium-consensus/src/Concordium/Scheduler.hs

```diff
@@ -2077,280 +2077,292 @@ handleConfigureBaker
         withDeposit wtc tickGetArgAndBalance chargeAndExecute
       where
         senderAccount = wtc ^. wtcSenderAccount
+        senderAccountIndex = fst senderAccount
+        bid = BakerId senderAccountIndex
         senderAddress = wtc ^. wtcSenderAddress
-        configureAddBakerArg =
```

Comment:
Why do we have almost complete rewrites of these functions induced by the cooldown changes?